dns/bind/src/opnsense/service/conf/actions.d/actions_bind.conf

```diff
@@ -22,6 +22,12 @@ parameters:
 type:script_output
 message:request BIND status
 
+[reload]
+command:/usr/local/etc/rc.d/named reload
```

Comment:
I would reintroduce the status variant as it really seems to hang when bind is no running